.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This file feels like it doesn't really belong, since we're re-initing on travis

Reply:
This file acts a bit like an package-lock.json; it helps ensure that even on an init we consistently use the same provider version. [Terraform are clear that it should be checked in](https://www.terraform.io/docs/language/dependency-lock.html), and [I found this discussion interesting](https://www.reddit.com/r/Terraform/comments/m1j5oh/terraformlockhcl_realworld_purpose_vs_overhead/) for more info